docs/error_knowledge_base.md

```diff
@@ -418,14 +419,36 @@ See also: [Submitting a Package](how_to_add_packages.md#submitting-a-package).
 The recipe missess [short_paths](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths) attribute.
 It may happen due to the very long paths within source, build or package directories during the package creating.
 Consider adding the following code:
+
 ```python
 class SomeRecipe(ConanFile):
     ...
 
     short_paths = True
 ```
 
-See also: (Maximum Path Length Limitation)[https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd].
+See also: [Maximum Path Length Limitation](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd).
+
+#### **<a name="KB-H068">#KB-H068</a>: "TEST_TYPE MANAGEMENT"**
+
+https://github.com/conan-io/hooks/commit/ad867ad0a3656fe77ecc964e9b8f3657995f0d5d
+
+#### **<a name="KB-H069">#KB-H069</a>: "TEST PACKAGE - NO DEFAULT OPTIONS"**
+
+https://github.com/conan-io/hooks/commit/7349d593d0728ae665dcb2c92ed3038b3cbb8ee5
```

Comment:
**KB-H069**: No `options` or `default_options` should be listed in `test_package`. Basically what we want here is to test the package we have just built. Modifying the value of any option can modify the graph and the result is that we will be testing something different (maybe resulting in missing_packages).

Reply:
```suggestion
This is to ensure the exact package that is built and uploaded is tested against. When `options` of `default_options` are modified in a `test_package` it can possibly result in the graph being modified. The objective is to enforce quality of the packages and to avoid confusing "missing packages" errors.
```